model-optimizer/extensions/middle/FusedBatchNormTraining.py

```diff
@@ -44,15 +46,14 @@ class FusedBatchNormTraining(MiddleReplacementPattern):
     def pattern(self):
         return dict(
             nodes=[
-                ('op', dict(kind='op', op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3'],
-                 is_training=True))],
+                ('op', dict(kind='op', op=lambda op: op in ['batchNormTraining']))],
             edges=[]
         )
 
     def replace_pattern(self, graph: Graph, match: dict):
-        node = match['op']
-        node.is_training = False
-
+        old_node = match['op']
+        node = castBatchNormTraining2Inference(old_node)
+        old_node.replace_node(node)
```

Comment:
Please, do not use `replace_node` method. Instead, create a new `batchNormInference` node and connect data to its inputs (5) and **just one output**.